src/giskardpy/constraints.py

```diff
@@ -937,8 +937,35 @@ def update_god_map(self, identifier, updates):
                 self.get_god_map().safe_set_data(next_identifier, value)
             else:
                 self.update_god_map(next_identifier, value)
+                
+    def get_constraint(self):
+        return {}
 
+class UpdateYaml(Constraint):
 
+    def __init__(self, god_map, updates):
+        super(UpdateYaml, self).__init__(god_map)
+        self.update_yaml([], updates)
+
+    def update_yaml(self, identifier, updates):
+        if not isinstance(updates, dict):
+            raise GiskardException(u'{} used incorrectly, {} not a dict or number'.format(str(self), updates))
+        for member, value in updates.items():
+            if len(identifier) == 0 and member in self.get_god_map().safe_get_data(identifier)[u'rosparam'].keys():
```

Comment:
Can you please try to work with `identifier.rosparam` instead of the hardcoded string? This would make future refactoring easier.